data/journal_check/bug_refs.json

```diff
@@ -388,5 +388,26 @@
             "sle-micro": [ "5.3" ]
         },
         "type": "bug"
+    },
+    "containers/myapp-httpd-prefork-jqdn": {
+        "description": "myapp.*httpd-prefork: Could not reliably determine the server's fully qualified domain name",
+        "products": {
+            "alp":  ["0.1"]
+        },
+        "type": "ignore"
+    },
+    "containers/myapp-get-200": {
+        "description": "myapp.*GET /.*200 -",
+        "products": {
+            "alp":  ["0.1"]
+        },
+        "type": "ignore"
+    },
+    "containers/hello-kubic-pod-kubic-started": {
+        "description": "hello-kubic-pod.*Hello-Kubic.*started",
+        "products": {
+            "alp":  ["0.1"]
+        },
+        "type": "ignore"
     }
```

Comment:
Are all of these messages warnings or even errors ?!

Reply:
No, they are `info` messages from containers:
```
Aug 08 15:58:51.733399 localhost.localdomain myapp[3562]: 10.88.0.1 - - [08/Aug/2022 15:58:51] "GET / HTTP/1.1" 200 -
```
and
```
Aug 08 16:00:57.878292 localhost.localdomain hello-kubic-pod-0-hello-kubic[7137]: 2022/08/08 16:00:57 Hello-Kubic 1.4 started
```
and
```
Aug 08 16:04:14.817806 localhost.localdomain myapp[9633]: AH00558: httpd-prefork: Could not reliably determine the server's fully qualified domain name, using 10.0.2.100. Set the 'ServerName' directive globally to suppress this message
```

Reply:
That is weird, as the `journalctl --no-pager --quiet -p err -o short-precise` should print errors or higher messages. I guess that should be fixed somewhere if they are info messages.

Reply:
Hmmm, yes, that's weird but where? They are all from podman container so it will be probably related. But how? … :(

Reply:
@dcermak can you take a look please?

Reply:
@pdostal I really need some context here, I must confess that I have my troubles finding out what the issue here is.

Reply:
@dcermak we are checking journal for error logs and some messages from podman appear there. But it does not make any sense to me why `Hello-Kubic 1.4 started` is treater as error message.

Reply:
It seems to me like some containers are outputting everything to stderr.

Reply:
```bash
# podman run registry.opensuse.org/opensuse/leap:latest sh -c 'echo test345-OUT > /dev/stdout'
test345-OUT
# podman run registry.opensuse.org/opensuse/leap:latest sh -c 'echo test345-ERR > /dev/stderr'
test345-ERR
# journalctl --no-pager -b --quiet -p err -o short-precise | grep 'test345'
Aug 09 19:16:56.342123 podman pensive_euclid[5017]: test345-ERR
# journalctl --no-pager -b --quiet -o short-precise | grep 'test345'
Aug 09 19:16:47.501007 podman wonderful_bardeen[4823]: test345-OUT
Aug 09 19:16:56.342123 podman pensive_euclid[5017]: test345-ERR
```

Reply:
So the HTTP 200 entry is from our testing Python app and it's printed by `http.server` to stderr by default:
```
$ cat www.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# Summary: Simple http server replying a static page with CONTENT
#          By default it listens on *:80

from http.server import HTTPServer, BaseHTTPRequestHandler
import os

# Content of the static http page
CONTENT = "<html>The test shall pass</html>"


class TestPassRequestHandler(BaseHTTPRequestHandler):
    """
    Actual webserver request handler. It returns CONTENT on a GET request
    """

    def do_GET(self):
        self.send_response(200)
        self.end_headers()
        self.wfile.write(CONTENT.encode("UTF-8"))
    #def log_request(self, format, *args):
    #    return


# Main program routine: Setup and run the webserver
if __name__ == "__main__":
    print("TEST\n")
    # If defined, take the PORT and ADDR environment variables
    port, addr = int(os.environ.get("PORT", 8080)), os.environ.get("ADDR", "0.0.0.0")
    print("http serving on %s:%d" % (addr, port))
    httpd = HTTPServer((addr, port), TestPassRequestHandler)
    httpd.serve_forever()
pavel@pdostal-server:/data/openqa/tests/opensuse/data/containers alp_containers_enable'$ python3 www.py 1> /dev/null
127.0.0.1 - - [10/Aug/2022 08:17:01] "GET / HTTP/1.1" 200 -
```

I now think that there is no error in journal, neither in podman or conmon.
I will try to fix those messages one by one so the `bug_refs.json` will remain unchanged.

Reply:
I opened thkukuk/hello-kubic#1 for the `hello-kubic` stderr message.